python/tvm/relay/frontend/tensorflow.py

```diff
@@ -425,6 +427,106 @@ def _impl(inputs, attr, params):
         return out
     return _impl
 
+
+#Dilation2d
+def _dilation2d(opname):
+
+    def _impl(inputs, attr, params):
+        attr['data_format'] = 'NHWC'
```

Comment:
why does the layout hard-coded here?

Reply:
As per the TF 1.14 , there is no separate attribute data_format for operator dilation2d, so it was not passed /processed in attribute list. but for further processing we needed this, that is why we have introduced it.

Reply:
> why does the layout hard-coded here?
Hello Yong, not sure if you got to see my response. could you please confirm , if comment is fine?